### ISSUE TITLE
Server error when trying to initiate a new site collection using provisioning template

### ISSUE THREAD

**Issue by J-ess-e** [SharePoint/sp-provisioning-service#29]
**Category**

[ ] Question
[ ] Typo
[X ] Bug
[ ] Additional article idea

**Expected or Desired Behavior**
Provision site using the provisioning service templates.

**Observed Behavior**
Getting error when trying to provision a site collection using templates:
Server Error in '/tenant' Application.
This type of page is not served.
Description: The type of page you have requested is not served because it has been explicitly forbidden.     Please review the URL below and make sure that it is spelled correctly.

Requested URL: /tenant/home/provision

**Steps to Reproduce**
Click on a template withing the provisioning service site, and click on "Add to your tenant."

**Comment by J-ess-e** [comment]
![Provisioing_Service_Server_Error_2019-06-04_13-57-27](https://user-images.githubusercontent.com/45435455/58906123-d9636480-86d0-11e9-93e1-2fdc5ecd3b4b.png)

**Comment by PaoloPia** [comment]
Hi @J-ess-e,
Thanks for your feedback. Could it be that you experienced a temporary issue? We cannot reproduce it. Are you still experiencing it? Thanks.

**Comment by J-ess-e** [comment]
I apologize for the late reply. I am still experiencing this issue, and can provide a fiddler trace if that helps. It looks like it occurs right after it tries to get the security token.

**Comment by PaoloPia** [comment]
Yes please, eventually share it privately with me using my email paolo at pialorsi dot com. Thanks.

**Comment by Trevor-SDG** [comment]
Rather than open a ticket (unless that is preferred), I'm having the same issue trying to add any template.
![2019-08-01_12-29-22](https://user-images.githubusercontent.com/36713715/62317998-0d040600-b458-11e9-8694-f4ee6f20d3b3.png)
